com.nameslowly.coinauctions.server/src/main/resources/application.yml

```
@@ -11,4 +11,38 @@ spring:
+  metrics:
+    export:
+      prometheus:
+        enabled: true
+
+  ######################## 집킨 설정 ########################
+  zipkin:
+    tracing:
+      endpoint: "http://localhost:9411/api/v2/spans" # Zipkin's HTTP endpoint
+  tracing:
+    sampling:
+      probability: 1.0
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)